build-logic/convention/src/main/kotlin/GravatarAndroidApplicationConventionPlugin.kt

```diff
@@ -39,15 +39,47 @@ class GravatarAndroidApplicationConventionPlugin : Plugin<Project> {
         }
     }
 
-    private fun ApplicationExtension.configureBuildTypes() {
+    private fun ApplicationExtension.configureBuildTypes(project: Project) {
+        val secretsProperties = project.property("secretsProperties") as java.util.Properties
+        val secretsPath = project.property("secretsPath") as String
+        val canSignRelease = secretsProperties.isNotEmpty()
+
+        if (!canSignRelease) {
+            project.logger.warn("Release signing configuration skipped: no secrets properties found")
+        }
+
+        signingConfigs {
+            maybeCreate("debug").apply {
+                storeFile = project.rootProject.file("debug.keystore")
+                storePassword = "android"
+                keyAlias = "androiddebugkey"
+                keyPassword = "android"
+            }
```

Comment:
Is it necessary to share a debug keystore? If not, then I'd vote to not do this - less code to maintain.

I know that in other projects we do this, but I don't see a valid reason to justify it.

Reply:
thanks!